src/main/kotlin/kweb/ElementCreator.kt

```diff
@@ -31,15 +31,16 @@ open class ElementCreator<out PARENT_TYPE : Element>(
 
     companion object : KLogging()
 
-    private val cleanupListeners: MutableCollection<Cleaner> = ConcurrentLinkedQueue<Cleaner>()
+    private var cleanupListeners: MutableCollection<Cleaner>? = null
```

Comment:
Should this be volatile?

Reply:
That's a good question! Possibly yes, this can probably lead to concurrency issues in the current implementation, though it'd definitely introduce some overhead to volatile this. Would you like me to PR an update?

Reply:
If you wouldn't mind.

Reply:
Done in https://github.com/kwebio/kweb-core/pull/232